[2024-11-03T21:55:11.805+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: DAG_feature_selcet_and_data_augmentation.feature_selection_task manual__2024-11-03T21:55:09.600017+00:00 [queued]>
[2024-11-03T21:55:11.810+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: DAG_feature_selcet_and_data_augmentation.feature_selection_task manual__2024-11-03T21:55:09.600017+00:00 [queued]>
[2024-11-03T21:55:11.810+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-11-03T21:55:11.810+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-11-03T21:55:11.810+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-11-03T21:55:11.821+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): feature_selection_task> on 2024-11-03 21:55:09.600017+00:00
[2024-11-03T21:55:11.831+0000] {standard_task_runner.py:55} INFO - Started process 126 to run task
[2024-11-03T21:55:11.835+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'DAG_feature_selcet_and_data_augmentation', 'feature_selection_task', 'manual__2024-11-03T21:55:09.600017+00:00', '--job-id', '85', '--raw', '--subdir', 'DAGS_FOLDER/feature_and_augm_dag.py', '--cfg-path', '/tmp/tmp98j5jos5']
[2024-11-03T21:55:11.838+0000] {standard_task_runner.py:83} INFO - Job 85: Subtask feature_selection_task
[2024-11-03T21:55:11.937+0000] {task_command.py:388} INFO - Running <TaskInstance: DAG_feature_selcet_and_data_augmentation.feature_selection_task manual__2024-11-03T21:55:09.600017+00:00 [running]> on host b77d411e2e76
[2024-11-03T21:55:11.966+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=House_Price_Prediction Team
AIRFLOW_CTX_DAG_ID=DAG_feature_selcet_and_data_augmentation
AIRFLOW_CTX_TASK_ID=feature_selection_task
AIRFLOW_CTX_EXECUTION_DATE=2024-11-03T21:55:09.600017+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-11-03T21:55:09.600017+00:00
[2024-11-03T21:55:11.970+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/feature_and_augm_dag.py", line 33, in feature_selection_callable
    raise ValueError("No encoded data found in XCom for 'encoded_data'")
ValueError: No encoded data found in XCom for 'encoded_data'
[2024-11-03T21:55:11.973+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=DAG_feature_selcet_and_data_augmentation, task_id=feature_selection_task, execution_date=20241103T215509, start_date=20241103T215511, end_date=20241103T215511
[2024-11-03T21:55:11.978+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 85 for task feature_selection_task (No encoded data found in XCom for 'encoded_data'; 126)
[2024-11-03T21:55:12.017+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-11-03T21:55:12.029+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
